internal/librarian/generate.go

```diff
@@ -85,6 +85,7 @@ func init() {
 	addFlagAPI(fs, cfg)
 	addFlagBuild(fs, cfg)
 	addFlagHostMount(fs, cfg)
+	addFlagPushConfig(fs, cfg)
```

Comment:
iiuc, #805 should include adding functionality of this flag to generate and related testing?